### DISCUSSION TITLE
I cannot for the life of me figure Loras out for colab.

### DISCUSSION THREAD

**Discussion by Achroous** [lllyasviel/Fooocus#2756] [Q&A] [answered]
I've looked everywhere, and it says to use gitpull, but I cant figure it out. Am I supposed to make a new cell? Put it in with the existing one? I'm so tech illiterate that I cant understand half of what people say, so if someone could dumb it down, I'd appreciate it. Sorry for the probably stupid question.

**Comment by mashb1t** [comment, marked as answer]
You need to add one line to make the LoRA file available in Colab. You can either add/modify a preset to download the LoRA or add a direct download to the Colab commands before Fooocus execution.
Also see here: https://github.com/lllyasviel/Fooocus/issues/1528#issuecomment-1865288435

**Reply by Achroous** [reply]
You are amazing. Thank you so much!
